templates/cli/lib/commands/generators/databases.ts

```
@@ -0,0 +1,401 @@
+        break;
+      case "integer":
+      case "double":
+        type = "number";
+        break;
+      case "boolean":
+        type = "boolean";
+        break;
+      case "relationship":
+        const relatedCollection = collections.find(
+          (c) => c.$id === attribute.relatedCollection,
+        );
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)